source/dub/data/settings.d

```diff
@@ -17,6 +17,7 @@ public enum SkipPackageSuppliers {
     standard,   /// Does not use the default package suppliers (`defaultPackageSuppliers`).
     configured, /// Does not use default suppliers or suppliers configured in DUB's configuration file
     all,        /// Uses only manually specified package suppliers.
+    default_,   /// The value wasn't specified. It is provided in order to know when it is safe to ignore it
```

Comment:
Instead of modifying the enum, passing a `Nullable` would be a better option, wouldn't it ?

Reply:
When I wrote this I considered both passing a Nullable in the constructor or modifying the enum as possible solutions. My final conclusion was that I would rather not change the constructor semantics and, instead, modify the enum.

I wouldn't mind doing as you say and changing the approach to use a Nullable.